Mathlib/Data/Matrix/Basic.lean

```diff
@@ -1677,6 +1677,9 @@ def mulVec [Fintype n] (M : Matrix m n α) (v : n → α) : m → α
 @[inherit_doc]
 scoped infixr:73 " *ᵥ " => Matrix.mulVec
 
+lemma mulVec_def [Fintype n] (M : Matrix m n α) (v : n → α) :
```

Comment:
Is it really useful? Can you point at a place where this is needed.

Reply:
In the first step of the proof of Siegel's lemma one needs to unfold the definition of mulVec. I thought the use of `unfold` was deprecated and one is supposed to write lemmas like that one.